<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Proyectos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cProyecto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bBusqueda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bAnadir">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir Proyecto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bEditar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSalir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Atrás</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lNombre">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de proyecto</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cTipo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lTipo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de proyecto</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSeleccion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccion de proyecto</string>
    </property>
   </widget>
   <widget class="QLabel" name="lRequisitos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Requisitos de Participacion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lDescripcion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="eRequisitos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>591</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="eDescripcion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>591</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bActualizar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar Vista</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
